api/requirements.txt

```diff
@@ -0,0 +1,3 @@
+flask
```

Comment:
Is there a way in zappa to host the api behind a http service locally (like in chalice)?

Reply:
`flask run`

Reply:
`flask run`